.github/workflows/typo3_v11ci.yml

```diff
@@ -0,0 +1,38 @@
+# Adapted from https://github.com/TYPO3GmbH/blog/blob/master/.github/workflows/ci.yml
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        typo3: [^11.5]
+        php: ['7.4', '8.0']
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup PHP ${{ matrix.php }}, with composer and extensions
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: mbstring, dom, zip, mysql
```

Comment:
I can only find linting and php-cs-fixer. No func. test. So mysql is not needed here

Reply:
Done